validator/docs/schemas.md

```diff
@@ -0,0 +1,195 @@
+
+**_Note_**: All the variables should be prefixed by `tpl`. To avoid colliding with the 
+
+```yaml
+templateVars:         # Variables used in conf templates
+  tplCriticity:       # Define if the app is on the critical path, STRING - high, low, none
+  tplPublicServices:  # Define if the application web is public accessible or not, BOOLEAN
+  tplExpandedTimeout: # Define the timeout on the ingress controller, STRING
+  tplSessionAffinity: # Activate or deactivate session affinity on ingress controller, BOOLEAN
+  tplTerminationGracePeriod: # Define POD termination grace period, INTEGER
+  tplCanary:          # Define name for canary, STRING
+```
+
+### Dependencies
+
+This section can be used when your application depends or makes use of another applications. In this section you can specify the list dependencies used by your project. 
+
+```yaml
+dependencies:
+  - api-v1
+  - voice-service
+  - payment-service
+  # .. and more
+  # if your application does not have dependencies, you can omit this section
+```
+
+### Public
+Determines if the project is publicly available for other services in your kubernetes namespace, this will allow internal interaction between applications.
+
+```yaml
+public: true 
+```
+
+### Public Aliases
+In this section allows at pod-level to override the hostname resolution when DNS and other options are not applicable. This is a feature of Kubernetes, that allows to reach your application with a more friendly name.
+
+You can define an alias for your application to facilitate public access.
+
+If you need further documentation of how DNS and the network layer works in Kubernets. We recommend you to read [the official Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/)
+
+```yaml
+public_aliases:
+  - www # STRING
+  - api # STRING
+```
+
+### Probes
+
+In this section, you can define some Kubernetes' probes for the web process. It is currently not possible to define probes for other processes.
+
+```yaml
+probes:
+  web:
+    liveness:
+      delay: 20         # NUMBER
+      path: /heartbeat  # STRING
+```
+
+**_Note_**: If you need more documentation on what probes are and their usage, visit the [official Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes).
+
+### Variables
+There are two categories of environment variables that you can add to your application: `ope` and `app`. 
+
+The idea is to separate concerns between the variables that your application needs.
+
+- `ope`: These are used to specify operational values such as URLs, certificates, auth public keys, connection strings, integration with third party services and other services information used by your app.
+- `app`: These are the variables used internally for adding logic to the application, such as: fixed values, timeouts, default values, feature flags, ids, and more.
+
+
+Environment variables used in the application.
```

Comment:
This sentence look weird (and alone 😢). Did you perhaps forget to remove it after writing the paragraph above ?